### ISSUE TITLE
BubbleUPnP only discovered when it is starting up

### ISSUE THREAD

**Issue by sixtyfive** [masmu/pulseaudio-dlna#275]
In total, I've four Android phones I use as DLNA renderers in diffferent rooms. Three of them work just fine, i.e. the have BubbleUPnP set to automatically start when they boot and everything except for the Local Renderer is disabled. The fourth phone is setup the same way (as far as I can tell) but only gets discovered if pa-dlna is started *first*, and then BubbleUPnP is *re*started.

Is there any way to find out what might be different about the fourth phone? (Hardware-wise, they're all different: Galaxy S1, Moto X / ghost, HTC One / m7 and the problematic one is a Nexus 5).

**Comment by masmu** [comment]
@sixtyfive Which version are you using of pulseaudio-dlna? I am asking because there is some updated network code in current master.
- Fixed a bug where the detection of DLNA devices failed when there were multiple network interfaces
- The application now binds to all interfaces by default

How does your network setup look like? Switches? Wifi? Cable? It there maybe a network repeater involved?
Does your computer have multiple network interfaces?

**Comment by sixtyfive** [comment]
Hi @masmu! Sorry, I should have mentioned that I `git pull`'d before posting the issue, so it's today's master.

The network setup is very simple: a WiFi router to which all four phones as well as my laptop are connected. All IP addresses are inside of 10.0.0.0/24 and all devices can ping each other.

With BubbleUPnP running only on the Nexus, upon pa-dlna startup, the last two messages are:

```
02-28 16:19:42 pulseaudio_dlna.pulseaudio   INFO     Could not get default sink. Perhaps there is no one set?
02-28 16:19:45 pulseaudio_dlna.discover   INFO     SSDPDiscover.search()
```

Restarting BubbleUPnP on the Nexus causes another message:

```
          INFO     Added the device "Nexus 5 (DLNA)".
```

What other debugging could I do?

**Comment by sixtyfive** [comment]
Now this is weird. There are no wifi power saving settings available on the Nexus o_O, even though it's running a custom ROM (Cyanogen 14.1). I think I'll chalk this up to be the ROM's fault then, sorry for bothering you with it!

**Comment by sixtyfive** [comment]
Hmm. So I'm back to stock, so it wasn't the custom ROM's fault. The stock ROM also has all the usual energy saving settings, but the behaviour is the same as described, so re-opening...

**Comment by FranziskaBlunck** [comment]
checked 'dem black list/deep sleep of the stand by mode for unknown yet still required processes? ('twas my problem on a different issue)

**Comment by sixtyfive** [comment]
Definitely nothing to do with any power settings. Same thing if the phone is kept active while pa-dlna is restartet.

**Comment by FranziskaBlunck** [comment]
("t", "d")

**Comment by masmu** [comment]
Sorry for the delay... but I was pretty busy recently.

I just tested this myself. I am on current master and it works for me using a Nexus 5 (stock rom). I started _BubbleUPnP_ and after that I started _pulseaudio-dlna_. Also did that test several times and also did it using a fresh booted Android to exclude any side effects of running applications.

Just a quick side note: BubbleUPnP 2.8.7 has to following line in the changelog:
- fixed BubbleUPnP not listed anymore as a player in some third party apps

Is that problem still reproducible?
